codyze-core/src/main/kotlin/de/fraunhofer/aisec/codyze/Sarif.kt

```diff
@@ -104,6 +102,8 @@ fun QueryTree<*>.getCodeflow(): List<List<Node>> {
         this.children.flatMap { it.getCodeflow() }
     } else if (this.value is List<*>) {
         listOf((this.value as Iterable<*>).filterIsInstance<Node>())
+    } else if (this.value is NodePath) {
```

Comment:
Can/should this happen?

Reply:
Yes and it did, in the past here some results were returned as List<Node> and handled properly, now, in case a NodePath is returned we have to unwrap.

Reply:
This may be the result of me accidentally adding a NodePath to the value of a QueryTree where before we had a List<Node>